### ISSUE TITLE
Documentation on choosing between "equally appropriate" icons does not match the provided example

### ISSUE THREAD

**Issue by pwillcode** [w3c/manifest#940]
In [the documentation](https://github.com/w3c/manifest/blob/7e609816a3b39934ebb9605133e55c9d55bfca50/index.html#L1306) when describing how to choose between "equally appropriate" icons:
> ... a user agent MUST use the last one declared in order ...

However, the example immediately following seems to describe the opposite order:

> ... If the user agent doesn't support WebP, it falls back to the second icon of the same size. ... 
(The webp icon in the example is the first, not the last one declared in order.)

Reading this, it is unclear to me which is the intended order.

**Comment by aarongustafson** [comment]
Agreed. I could have sworn we actually had a discussion about this at one point and changed the behavior to match [HTML’s first-match policy for media](https://html.spec.whatwg.org/multipage/media.html#concept-media-load-algorithm) (e.g., `video`, `picture`) but I’m having a hard time finding that discussion in the issues. @marcoscaceres am I dreaming that up?
